grails-app/services/org/pih/warehouse/inventory/OutboundStockMovementService.groovy

```diff
@@ -100,6 +100,13 @@ class OutboundStockMovementService {
                     }
                 }
             }
+            if (stockMovement.approvers) {
+                requisition {
+                    approvers {
+                        'in'("id", stockMovement.approvers.collect { it.id })
```

Comment:
I didn't see it when I was writing my comment above, but anyway, to be sure nothing unexpected will happen, please do it as `it?.id`, because when having `null` in your List created from `getAll`, you would get an NPE here.